src/Tools/AI Test Toolkit/src/Logs/AITLogEntries.Page.al

```diff
@@ -68,6 +68,23 @@ page 149033 "AIT Log Entries"
                 {
                     StyleExpr = StatusStyleExpr;
                 }
+                field(Accuracy; Rec.Accuracy)
```

Comment:
we need to expose these in the API as well

Reply:
Feel free to create a separate work item, we can uptake the API changes in the same work item.